### PR TITLE
Updated .NET Core content for 2.1 release

### PR DESCRIPTION
**Pull request by ahardin-rh**
Preview Build: http://file.rdu.redhat.com/~ahardin/06042018/dot-net-core-2-1/using_images/s2i_images/dot_net_core.html

### CODE REVIEW

**Comment by jerboaa** [comment]
cc @tmds @aslicerh

File: using_images/s2i_images/dot_net_core.adoc [hunk 1]
```diff
@@ -41,18 +42,18 @@ ifdef::openshift-enterprise[]
 and {product-title} versions 3.3 and later
 endif::openshift-enterprise[]
 
-The .NET Core software collection for version 2.0 supports the
+The .NET Core software collection for versions 2.1 and 2.0 supports the
```
Comment (tmds) [inline using_images/s2i_images/dot_net_core.adoc]: Can you please put this paragraph below the next one? It has become less important.
You can remove the first sentence, so it is clear the paragraph only applies to Versions 1.0 and 1.1.

**Comment by tmds** [comment]
@ahardin-rh thanks for working on this. We are reviewing the '.NET Core Getting started' guide also. When that review is done, I'll check the table with the envvars matches with what we have here.

**Comment by tmds** [comment]
@ahardin-rh this is the table we have in the getting started. Can you copy-paste this format?
```
|**Variable Name** |**Description** |**Default**
|**DOTNET_STARTUP_PROJECT** |Used to select the project to run. This must be a project file (for example, `csproj` or `fsproj`) or a folder containing a single project file. |`.`
|**DOTNET_SDK_VERSION** |Used to select the default sdk version when building. If there is a `global.json` file in the source repository, that takes precedence. When set to latest the latest sdk in the image is used. | Lowest sdk version available in the image.
|**DOTNET_ASSEMBLY_NAME** |Used to select the assembly to run. This must not include the .dll extension. Set this to the output assembly name specified in `csproj` (PropertyGroup/AssemblyName). |The name of the `csproj` file
|**DOTNET_RESTORE_SOURCES** |Used to specify the space-separated list of NuGet package sources used during the restore operation. This overrides all of the sources specified in the `NuGet.config` file. |
|**DOTNET_TOOLS** |Used to specify a list of .NET tools to install before building the app. It is possible to install a specific version by post pending the package name with ``@<version>``. |
|**DOTNET_NPM_TOOLS** |Used to specify a list of NPM packages to install before building the application. |
|**DOTNET_TEST_PROJECTS** |Used to specify the list of test projects to test. This must be project files or folders containing a single project file. `dotnet test` is invoked for each item. |
|**DOTNET_CONFIGURATION** |Used to run the application in Debug or Release mode. This value should be either _Release_ or _Debug_. |`Release`
|**DOTNET_VERBOSITY** |Used to specify the verbosity of the dotnet build commands. When set, the environment variables are printed at the start of the build. This variable can be set to one of the msbuild verbosity values (`q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`).|
|**HTTP_PROXY, HTTPS_PROXY** |Configures the HTTP/HTTPS proxy used when building and running the application.|
|**NPM_MIRROR** |Use a custom NPM registry mirror to download packages during the build process. |
|**ASPNETCORE_URLS** |This variable is set to `http://*:8080` to configure ASP.NET Core to use the port exposed by the image. Changing this is not recommended. |`http://*:8080`
```

**Comment by ahardin-rh** [comment]
@tmds That format does not match our [guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc) (for example, how we format variable names) but I can cross-check everything and ensure the content matches. Is there anything that you see that is not correct? Thanks!

**Comment by ahardin-rh** [comment]
@tmds I cross-checked everything with the source material you provided, made updates, and applied our style guidelines. I also updated the preview build: http://file.rdu.redhat.com/%7Eahardin/06042018/dot-net-core-2-1/using_images/s2i_images/dot_net_core.html but you may need to refresh/clear your cache to see the latest changes reflected. Let me know if I missed anything. Thanks!

**Comment by leswilliams44** [comment]
Looks great, Ashley! Thanks for working on this. We aim to release on 6/14/2018.

File: using_images/s2i_images/dot_net_core.adoc [hunk 2]
```diff
@@ -111,55 +109,63 @@ available to the build steps.
...
 |`DOTNET_CONFIGURATION`
 |Used to run the application in `Debug` or `Release` mode. This value should be
 either `Release` or `Debug`.
 |`Release`
 
+|`DOTNET_VERBOSITY`
+|Used to specify the verbosity of the dotnet build commands. When set, the
+environment variables are printed at the start of the build. This variable can
+be set to one of the msbuild verbosity values (`q`[uiet], `m`[inimal], `n`[ormal],
+`d`[etailed], and `diag`[nostic]).
```
Comment (tmds) [inline using_images/s2i_images/dot_net_core.adoc]: nit: the long names are also accepted by the tooling, so we should include them in the backticks:
`q[uiet]`, `m[inimal]`, `n[ormal]`,
+`d[etailed]`, and `diag[nostic]`

**Review by tmds — APPROVED** [review]

**Comment by ahardin-rh** [comment]
@tmds Thanks! Nits addressed.

**Pull request merged**